ui/apps/platform/cypress/integration/vulnerabilities/exceptionManagement/cancelRequestFlow.test.ts

```diff
@@ -46,21 +46,61 @@ describe('Exception Management Request Details Page', () => {
     });
 
     it('should be able to cancel a request if the user is the requester', () => {
+        deferAndVisitRequestDetails({
+            comment,
+            expiry,
+            scope,
+        });
         cy.get('button:contains("Cancel request")').click();
         cy.get('div[role="dialog"]').should('exist');
         cy.get('div[role="dialog"] button:contains("Cancel request")').click();
         cy.get('div[role="dialog"]').should('not.exist');
         cy.location().should((location) => {
             expect(location.pathname).to.eq(pendingRequestsPath);
         });
+        cy.get('table tbody tr').should('not.exist');
     });
 
     it('should be able to see how many CVEs will be affected by a cancel', () => {
+        deferAndVisitRequestDetails({
+            comment,
+            expiry,
+            scope,
+        });
         cy.get('table tbody tr:not(".pf-c-table__expandable-row")').then((rows) => {
             const numCVEs = rows.length;
             cy.get('button:contains("Cancel request")').click();
             cy.get('div[role="dialog"]').should('exist');
             cy.get(`div:contains("CVE count: ${numCVEs}")`).should('exist');
         });
     });
+
+    it('should not see a cancelled request in the approved deferrals table', () => {
+        deferAndVisitRequestDetails({
+            comment,
+            expiry,
+            scope,
+        });
+        approveRequest();
+        cy.get('button:contains("Cancel request")').click();
+        cy.get('div[role="dialog"]').should('exist');
+        cy.get('div[role="dialog"] button:contains("Cancel request")').click();
+        cy.get('div[role="dialog"]').should('not.exist');
```

Comment:
Minor nit if we revisit these tests in the near future, it might be beneficial to group this into a `cancelRequest()` helper.